### PR TITLE
Support for Layout Options on a Node

### PR DESCRIPTION
**Pull request by simonwarchol**
I'd like to add in the functionality discussed in https://github.com/cytoscape/cytoscape.js-elk/pull/33. 
I'm not totally sure what you meant in the comments of that PR to implement the change as a function instead of this approach, but I've changed the layout options to be stored in scratch instead of data.

This is critical functionality.

### CODE REVIEW

**Comment by simonwarchol** [comment]
Hi @maxkfranz,
I've updated the PR accordingly.
-Simon

File: README.md [hunk 1]
```diff
@@ -86,6 +87,12 @@ var options = {
 cy.layout( options ).run();
 ```
 
+You can set layout options per node by defining a `nodeLayoutOptions` function which is called on a per-node basis. This is useful for tweaking the layout of a particular node, like for [setting its partition](https://www.eclipse.org/elk/reference/options/org-eclipse-elk-partitioning-partition.html) for the layered layout.
+For instance, if you want to store these options within the node's `scratch` object, you can do something like this:
+
+```js
+nodeLayoutOptions: (n: any) => n.scratch('layoutOptions')
```
Comment (maxkfranz) [inline README.md:94]:
```suggestion
nodeLayoutOptions: node => n.scratch('layoutOptions')
```

Comment (maxkfranz) [reply README.md:94]: The consumer might not be using TS.  What do you think about a change like this?

**Review by maxkfranz — COMMENTED** [review]
This looks good.

There's the point re. the documentation to consider.  With that addressed, we should move forward with releasing this.

**Comment by simonwarchol** [comment]
Hi @maxkfranz ,
Yep, i like that. Let me know if there's anything else that I can do.
-Simon

**Comment by maxkfranz** [comment]
Great.  Merged and published as 2.2.0.

All the best with your project, @simonwarchol.

**Pull request merged**